tiered-storage/jcloud/src/main/java/org/apache/bookkeeper/mledger/offload/jcloud/impl/BlobStoreBackedInputStreamImpl.java

```diff
@@ -73,7 +87,12 @@ private boolean refillBufferIfNeeded() throws IOException {
                                      objectLen - 1);
 
             try {
+                long startReadTime = System.nanoTime();
                 Blob blob = blobStore.getBlob(bucket, key, new GetOptions().range(startRange, endRange));
+                if (this.mxBean != null) {
```

Comment:
When will mxBean be null ?

Reply:
it should not be null, there is a check in case of cause exception